### PR TITLE
Fix for issue #213

### PR DESCRIPTION
**Pull request by ankhoa1212**
To fix issue mit-han-lab/torchquantum#213, it looks like the ```QuantumDevice``` needs to be initialized with the batch size (batch size is 1 by default which is why the size was [1, 2] instead of [256, 2]). 

After this change, there is a ```NameError``` for ```CliffordQuantizer```:
image
which can be fixed by importing it in the ```torchquantum/operator/op_types.py``` file, which calls the ```CliffordQuantizer``` function.

After these changes, I believe ```examples/clifford_qnn/mnist_clifford_qnn.py``` can run out-of-the-box:
clifford_success
@Hanrui-Wang @01110011011101010110010001101111

### CODE REVIEW

**Review by 01110011011101010110010001101111 — CHANGES_REQUESTED** [review]
Thanks for figuring this out! Just a quick question on the extra import then I can merge this in!

File: torchquantum/operator/op_types.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from ..macro import C_DTYPE, F_DTYPE
 from typing import Iterable, Union, List
 from enum import IntEnum
+from torchquantum.util.quantization.clifford_quantization import CliffordQuantizer
```
[the remark this answers is not in the document]
Comment (01110011011101010110010001101111) [reply torchquantum/operator/op_types.py:10]: Oh, wait, I missed your initial comment.

[the remark this answers is not in the document]
Comment (01110011011101010110010001101111) [reply torchquantum/operator/op_types.py:10]: Can you instead import this lazily, so in the conditional branch where we actually call `CliffordQuantizer`? Afterward, will merge in!

**Review by 01110011011101010110010001101111 — CHANGES_REQUESTED** [review]

**Comment by 01110011011101010110010001101111** [comment]
Also, don’t worry about tests, it’s likely from numpy 2.0 breaking. Once we merge, I think we should be good.

**Comment by ankhoa1212** [comment]
Alright I moved the import into the conditional branch, so it should be good now.

**Comment by 01110011011101010110010001101111** [comment]
Merged, thanks so much!

**Pull request merged**